server/routes/availability.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const validateSitter = require("../middleware/validateSitter");
+const {
+  getSchedules,
+  setScheduleActive,
+  getScheduleById,
+  getActiveSchedule,
+  addSchedule
+} = require("../controllers/availability");
+
+router.route("/").get(protect, validateSitter, getSchedules);
+router.route("/active").get(protect, validateSitter, getActiveSchedule);
+router.route("/:scheduleId").get(protect, validateSitter, getScheduleById);
+router.route("/").post(protect, validateSitter, addSchedule);
```

Comment:
Hi @ezeilo-su , 

Thank you so much for your reviewing! I actually would like to ask questions regarding correctly linking pr to the correct issue you talked below. Since each time I link I clicked connect in the Pr as shown below:
![Screen Shot 2022-02-05 at 11 08 23 AM](https://user-images.githubusercontent.com/19254360/152649291-3ed64f24-1176-4de3-ab06-75c1fc1a8055.png)
Are there any additional actions I am supposed to add? Thank you!